GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
-                    <property name="homogeneous">True</property>
-                </packing>
-            </child>
-            <child>
-              <object class="GtkMenuToolButton" id="open_parents">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-                <property name="label" translatable="yes">Open parent task</property>
-                <property name="stock_id">gtk-go-up</property>
-                <accelerator key="p" signal="clicked" modifiers="GDK_CONTROL_MASK"/>
-                <child internal-child="accessible">
-                  <object class="AtkObject" id="open_parents-atkobject">
-                    <property name="AtkObject::accessible-name" translatable="yes">Open parent task</property>
-                    <property name="AtkObject::accessible-description" translatable="yes">Opens parent task of this task</property>
-                  </object>
-                </child>
-                <signal name="clicked" handler="on_open_parent_clicked" swapped="no"/>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="homogeneous">True</property>
-              </packing>
-            </child>
-            <child>
-              <object class="GtkSeparatorToolItem" id="separator_note">
-                <property name="visible">True</property>
-                <property name="can_focus">False</property>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="homogeneous">True</property>
-              </packing>
-            </child>
+            <property name="sensitive">True</property>
+            <property name="label" translatable="yes">Mark as Done</property>
+            <property name="tooltip_text" translatable="yes">Mark the task as done</property>
+            <property name="valign">center</property>
+            <signal handler="mark_as_done_clicked" name="clicked" swapped="no"/>
           </object>
           <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">0</property>
+            <property name="pack_type">start</property>
           </packing>
+          <style>
+            <class name="text-button"/>
+          </style>
         </child>
         <child>
-          <object class="GtkBox" id="box3">
+          <object class="GtkMenuButton" id="editor_menu">
             <property name="visible">True</property>
-            <property name="can_focus">False</property>
+            <property name="sensitive">True</property>
+            <property name="tooltip_text" translatable="yes">Task action menu</property>
+            <property name="valign">center</property>
+            <property name="popover">menu_popover</property>
```

Comment:
it might be better to name it `editor_menu_popover` as you will have more popover menus